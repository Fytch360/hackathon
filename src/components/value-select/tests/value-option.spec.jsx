import React from 'react'
import { shallow } from 'enzyme'

import { ValueOption } from '..'

describe('<ValueOption />', () => {

    test('Should be available', () => {
        expect(ValueOption).toBeDefined()
    })
    
    test('Snapshot', () => {
        const wrapper = shallow(
            <ValueOption
                size="lg"
                value="value-example"
                icon="icon:core/iconNameExample"
                title="title-example"
                description="description-example"
                additional="additional-title-example"
                additionalDescription="additional-description-example"
            />
        )

        expect(wrapper).toMatchSnapshot()
    })
})
