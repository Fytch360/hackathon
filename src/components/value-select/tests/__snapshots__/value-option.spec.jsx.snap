// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ValueOption /> Snapshot 1`] = `
<Fragment>
  <IconLoaderStyled
    name="icon:core/iconNameExample"
    onClick={[Function]}
    onError={[Function]}
    size="lg"
  />
  <ItemColumnStyled>
    <ItemTitleStyled
      ellipsis={true}
      fontWeight="medium"
      size="lg"
    >
      title-example
    </ItemTitleStyled>
    <ItemDescriptionStyled
      size="md"
    >
      description-example
    </ItemDescriptionStyled>
  </ItemColumnStyled>
  <ItemColumnStyled>
    <ItemAsideStyled
      fontWeight="medium"
      size="lg"
    >
      additional-title-example
    </ItemAsideStyled>
    <ItemAsideDescriptionStyled
      size="md"
    >
      additional-description-example
    </ItemAsideDescriptionStyled>
  </ItemColumnStyled>
</Fragment>
`;
